tutorials/tutorial_e_signature_integration.md

```diff
@@ -0,0 +1,111 @@
+
+### Subcontractor Invoices
+
+- /{project_id}/project/commitments/work_order_contracts/{word_order_id}/requisitions/{requisition_id}.pdf
+
+### Change Orders
+
+- /{project_id}/project/commitments/purchase_order_contracts/{purchase_order_contract_id}/change_orders/commitment_contract_change_orders/{change_order_package_id}.pdf
+- /{project_id}/project/commitments/work_order_contracts/{work_order_contract_id}/change_orders/work_contract_change_orders/{change_order_package_id}.pdf
+- /{project_id}/project/prime_contracts/{prime_contract_id}/change_orders/change_order_packages/{change_order_package_id}.pdf
+
+### RFIs
+
+- (All repsonses) /{project_id}/project/rfi/view_pdf.pdf?id={rfi_id}&only_official=false
+- (Official responses only) /{project_id}/project/rfi/view_pdf.pdf?id={rfi_id}&only_official=true
+
+### Submittals
+
+- /{project_id}/project/submittal_logs/{submittal_id}.pdf
+
+### Daily Log
+
+- /{project_id}/project/daily_log/list.pdf?date={date}
+
+## Documents Tool Integration
+
+Additional information on integrating with the Documents tool can be found at [developers.procore.com](https://developers.procore.com/documentation/tutorial-uploads).
+We recommended that executed documents be saved to a folder within the Procore Documents tool for reference.
+This tool allows for Admin-level users to make folders private. We also recommend this approach for all contract documents.
+
+## Additional Functionality
+
+Additional functionality can be added to the integration to support contract execution.
+Through Procore’s API, an integration can update the status of a contract to “Complete,” mark the contract as executed and record the executed date.
+
+**Note:** Procore allows *Custom Workflows* to be assigned with these tools.
+Custom Workflow in this scenario mostly refers to internal “controls” or “approvals” prior to something being sent out.
+This might add some complexity to the update as it requires a different API call.
+(See information below.)
+
+Here are links to the API documentation covering the update actions discussed above for contracts, change orders, and workflows in Procore:
+
+### Contracts
+
+- [Prime Contract](https://developers.procore.com/reference/rest/v1/prime-contracts#update-prime-contract)
+- [Commitments](https://developers.procore.com/reference/rest/v1/work-order-contracts#update-work-order-contract) (Subcontracts/Work Order)
+- [Commitments](https://developers.procore.com/reference/rest/v1/purchase-order-contracts#update-purchase-order-contract) (Purchase Order)
+
+### Change Orders
+
+- [General Change Order Information](https://developers.procore.com/documentation/tutorial-change-orders)
+- [Change Order](https://developers.procore.com/reference/rest/v1/change-order-packages#update-change-order-package)
+(Note: Best practice would be to use the APIs to pull an “Approved Change Order” into the technology partner software through webhooks.)
```

Comment:
Adding a new line after a bullet point in markdown breaks out of the list. Should this note be on the same line as the bullet point above?

Similar comment for the bullet point below.